shoot-client/network-connection.sh

```diff
@@ -90,9 +90,10 @@ echo "pull-filter ignore redirect-gateway" >> openvpn.config
 echo "pull-filter ignore route-ipv6" >> openvpn.config
 echo "pull-filter ignore redirect-gateway-ipv6" >> openvpn.config
 
+
 # enable forwarding and NAT
-iptables --append FORWARD --in-interface tun0 -j ACCEPT
```

Comment:
Any particular reason why you removed this rules?

Reply:
I just made it working with IPv6 and dropped IPv4 stuff :) We could also leave it in place!